File: Python3/프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.py
def solution(num_list):
    list_len = len(num_list)
    num_list += [num_list[list_len - 1] * 2 if num_list[list_len - 1] <= num_list[list_len - 2] else num_list[list_len - 1] - num_list[list_len - 2]]
    return num_list

File: C++/프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list) {
    int last = num_list.back();
    int nextLast = num_list[num_list.size() - 2];
    num_list.push_back(last > nextLast ? last - nextLast : last * 2);
    return num_list;
}